learnCSharp/AsyncTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace learnCSharp
+{
+    public class AsyncTest
+    {
+        public async Task wait10Sec()
```

Comment:
Convention in C# is to use PascalCasing method names, that is, capital first letter and each word afterwards. Additionally, it's better to use full words for more descriptive names. This might be changed to `Wait10Seconds`.

Reply:
See the other two methods in this class as well!